src/components/era_commons/ERACommons.jsx

```diff
@@ -165,23 +163,35 @@ export default function ERACommons(props) {
           {expirationCount >= 0 && (
             <div className="era-commons-id-value">
               <span data-cy="era-commons-id-value">{eraCommonsId}</span>
-              <button data-cy="era-delete-icon" className="era-delete-icon" type="button" onClick={deleteNihAccount}>
+              <AsyncActionButton
+                className="era-delete-icon"
+                onClick={deleteNihAccount}
+                onError={displayError}
+                data-cy="era-delete-icon"
+                style={{
+                  cursor: 'pointer',
+                  color: '#333',
+                  backgroundColor: 'white',
+                  border: 'none',
+                  padding: '0',
+                  minWidth: '10px',
+                }}
```

Comment:
Should this button disappear after use?

Reply:
The behavior it has now, which is to show a spinner while the action is happening, then disappear, looks like better UX to me:

https://github.com/user-attachments/assets/e47c69d1-02d1-4a08-a25d-7f0fca13c729